src/js/step.js

```diff
@@ -348,7 +348,8 @@ export class Step extends Evented {
 
     this.el.hidden = false;
 
-    this.tooltip.position();
+    // TODO figure out what the Popper v2 method is to force position recalculation
```

Comment:
It’s `instance.update()`, note that it returns a promise, so you can await for it if you need to